pandas/core/arraylike.py

```diff
@@ -97,6 +97,92 @@ def _arith_method(self, other, op):
 
     @unpack_zerodim_and_defer("__add__")
     def __add__(self, other):
+        """
+        Get Addition of DataFrame and other, column-wise.
+
+        Equivalent to ``DataFrame.add(other)``.
+
+        Parameters
+        ----------
+        other : scalar, sequence, Series, dict or DataFrame
+            Object to be added to the DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            The result of adding ``other`` to DataFrame.
+
+        See Also
+        --------
+        DataFrame.add : Add a DataFrame and another object, with option for index-
+            or column-oriented addition.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'height': [1.5, 2.6], 'weight': [500, 800]},
+        ...                   index=['elk', 'moose'])
+        >>> df
+                height	weight
+        elk	1.5	    500
+        moose	2.6	    800
+
+        Adding a scalar affects all rows and columns.
+
+        >>> df[['height', 'weight']] + 1.5
+                height	weight
+        elk	3.0	    501.5
+        moose	4.1	    801.5
+
+        Each element of a list is added to a column of the DataFrame, in order.
+
+        >>> df[['height', 'weight']] + [0.5, 1.5]
+                height	weight
+        elk	2.0	    501.5
+        moose	3.1	    801.5
+
+        Keys of a dictionary are aligned to the DataFrame, based on column names;
+        each value in the dictionary is added to the corresponding column.
+
+        >>> df + {'height': 0.5, 'weight': 1.5}
+                height	weight
+        elk	2.0	    501.5
+        moose	3.1	    801.5
+
+        When `other` is a :class:`Series`, the index of `other` is aligned with the
+        columns of the DataFrame.
+
+        >>> s1 = pd.Series([0.5, 1.5], index=['weight', 'height'])
+        >>> df[['height', 'weight']] + s1
+                height	weight
+        elk	3.0	    500.5
+        moose	4.1	    800.5
+
+        Even when the index of `other` is the same as the index of the DataFrame,
+        the :class:`Series` will not be reoriented. If index-wise alignment is desired,
+        :meth:`DataFrame.add` should be used with `axis='index'`.
+
+        >>> s2 = pd.Series([0.5, 1.5], index=['elk', 'moose'])
+        >>> df[['height', 'weight']] + s2
+                elk	    height	moose	weight
+        elk	NaN	    NaN	    NaN	    NaN
+        moose	NaN	    NaN	    NaN	    NaN
```

Comment:
this still doesn't look quite aligned, running this locally I see
```python
       elk  height  moose  weight
elk    NaN     NaN    NaN     NaN
moose  NaN     NaN    NaN     NaN
```

does it work to just copy-and-paste that, and then add 8 spaces to each line? If you use tab then it may not work exactly

Reply:
Unfortunately, that doesn't seem to work. The most recent commit contains the results copy/pasted from running locally, with 8 spaces added to the left. That seems to mess up header alignment. (Adding another 4 spaces to the header is a slight improvement, but doesn't seem to 100% fix the issue.)

Reply:
I stand corrected and am apparently just really bad at spacing. Most recent commit (hopefully) has correct alignment.